src/store/addressbooks.js

```diff
@@ -512,6 +512,34 @@ const actions = {
 		await context.commit('addContactToAddressbook', contact)
 		return contact
 	},
+
+	/**
+	 * Copy a contact to the provided addressbook
+	 *
+	 * @param {Object} context the store mutations
+	 * @param {Object} data destructuring object
+	 * @param {Contact} data.contact the contact to copy
+	 * @param {Object} data.addressbook the addressbook to move the contact to
+	 * @returns {Contact} the new contact object
+	 */
+	async copyContactToAddressbook(context, { contact, addressbook }) {
+		// init new contact & strip old uid
+		const vData = ICAL.stringify(contact.vCard.jCal).replace(/^UID.+/im, '')
+		const newContact = new Contact(vData, addressbook)
+
+		try {
+			const response = await contact.dav.copy(addressbook.dav)
```

Comment:
Maybe use contact.addressbook.dav.createVCard(vData) instead to force a different filename and avoid conflicts